utensor_cgen/ir/base.py

```diff
@@ -205,6 +206,7 @@ class uTensorGraph(IRBase, _NoShallowCopyMixin):
   _backend = attr.ib(default='', type=str)
   ops_info = attr.ib(factory=dict)
   topo_order = attr.ib(factory=list, init=False)
+  data_manager = None
```

Comment:
It should be `data_manager = attr.ib(default=None)`.
Or `data_manger` will be a shared class-level attribute.

Please take a look over these examples for `attrs`:
http://www.attrs.org/en/stable/examples.html